Lib/asyncio/events.py

```diff
@@ -605,12 +605,8 @@ def new_event_loop(self):
 
 
 # A TLS for the running event loop, used by _get_running_loop.
-class _RunningLoop(threading.local):
-    _loop = None
-    _pid = None
-
-
-_running_loop = _RunningLoop()
+_running_loop = threading.local()
+_running_loop._loop_pid = (None, None)
```

Comment:
I don't think that the attribute has to be private, since the _running_loop variable is already prive. Rename _loop_pid to loop_pid.

Reply:
@haypo renamed.